src/toolbar/toolbar.html

```diff
@@ -1,8 +1,9 @@
 <div id="toolbar">
     <img id="logo" src="assets/svg/scottlogic_logo.svg" />
     <toolbar-icon name="minimise" icon-click="toolbarCtrl.minimiseClick()"></toolbar-icon>
-    <toolbar-icon name="maximise" ng-show="!toolbarCtrl.maximised" icon-click="toolbarCtrl.maximiseClick()"></toolbar-icon>
-    <toolbar-icon name="normal_size" ng-show="toolbarCtrl.maximised" icon-click="toolbarCtrl.normalSizeClick()"></toolbar-icon>
+    <toolbar-icon name="summarise" icon-click="toolbarCtrl.summariseClick()"></toolbar-icon>
+    <toolbar-icon name="maximise" ng-show="!toolbarCtrl.maximised && !toolbarCtrl.summarised" icon-click="toolbarCtrl.maximiseClick()"></toolbar-icon>
+    <toolbar-icon name="normal_size" ng-show="toolbarCtrl.maximised && !toolbarCtrl.summarised" icon-click="toolbarCtrl.normalSizeClick()"></toolbar-icon>
```

Comment:
I think it's probably fine just with one `&&`, if there's more then it's probably worth doing